### ISSUE TITLE
What do I do wrong?

### ISSUE THREAD

**Issue by AngeloMetal** [PawelGorny/lostword#1]
The mnemonic: ``reward news divorce text local race patrol eager planet dish chief actress`` creates the following address: ``bc1qh3gygn44mhkshg0j6va5nzevj67ys6cjfgqtxe`` on ``m/84'/0'/0'/0/0``
My Config.txt is:

```
KNOWN_POSITION
bc1qh3gygn44mhkshg0j6va5nzevj67ys6cjfgqtxe
12
reward
news
divorce
?
local
race
patrol
eager
planet
dish
chief
actress
m/0/0'
```
I run it, but it doesn't find the address:

<img src="https://i.imgur.com/ZhVfGdr.png">

What do I do wrong?

**Comment by WanderingPhilosopher** [comment]
Author states, "Program works with BIP141/P2WPKH or BIP32/P2PKH Derivation Path."
Maybe it doesn't work with BIP84

**Comment by PawelGorny** [comment]
There was problem with BIP84 derivation path, now it is solved with 0.8.1.
About the configuration mentioned: the address is not hardened, so the configuration should be:
`KNOWN_POSITION
bc1qh3gygn44mhkshg0j6va5nzevj67ys6cjfgqtxe
12
reward
news
divorce
?
local
race
patrol
eager
planet
dish
chief
actress
m/0/0`

or even without the last line

**Comment by AngeloMetal** [comment]
Thanks, it works now. Issue is now closed.
